src/modules/src/crtp_commander.c

```diff
@@ -55,8 +103,19 @@ static void commanderCrtpCB(CRTPPacket* pk)
   if(pk->port == CRTP_PORT_SETPOINT && pk->channel == 0) {
     crtpCommanderRpytDecodeSetpoint(&setpoint, pk);
     commanderSetSetpoint(&setpoint, COMMANDER_PRIORITY_CRTP);
-  } else if (pk->port == CRTP_PORT_SETPOINT_GENERIC && pk->channel == 0) {
-    crtpCommanderGenericDecodeSetpoint(&setpoint, pk);
-    commanderSetSetpoint(&setpoint, COMMANDER_PRIORITY_CRTP);
+  } else if (pk->port == CRTP_PORT_SETPOINT_GENERIC) {
+    switch (pk->channel) {
+    case 0:
+      crtpCommanderGenericDecodeSetpoint(&setpoint, pk);
+      commanderSetSetpoint(&setpoint, COMMANDER_PRIORITY_CRTP);
+      break;
+    case 1: {
+        uint8_t metaCmd = pk->data[0];
+        if (metaCmd < nMetaCommands && (metaCommandDecoders[metaCmd] != NULL)) {
+          metaCommandDecoders[metaCmd](pk->data + 1, pk->size - 1);
+        }
+      }
+      break;
+    }
```

Comment:
Named constants makes it easier to understand what the cases are handling. Maybe something like 
```
switch (pk->channel) {
  case SET_SETPOINT_CHANNEL:
    // ...
    break;
  case META_COMMAND_CHANNEL:
    // ...
    break;
  default:
    // Do nothing
    break;
}
```

and a default for completeness.